modules/coreboot

```diff
@@ -24,6 +34,11 @@ else ifeq "$(CONFIG_COREBOOT_VERSION)" "4.14"
 	coreboot_hash := d907379b727561d7ddd1d80b2fabaa373db00c9805719116f591cbc948173c6e
 	coreboot-blobs_hash := 9ee2fe5ba37d0e214000b8655acf8922de6df792adb75790559db4c160847921
 	coreboot_depends := $(if $(CONFIG_PURISM_BLOBS), purism-blobs)
+else ifeq "$(CONFIG_COREBOOT_VERSION)" "talos_2_support_payload"
+	coreboot_version = git
+	coreboot_patch_version := talos_2_support_payload
+	coreboot_commit_hash = ce64c091228ed8878263263fd7fe700e64557148
```

Comment:
@SergiiDmytruk Can we use dasharo and latest commit from release branch?  https://github.com/Dasharo/coreboot/tree/raptor-cs_talos-2/release/

Reply:
Or maybe the develop branch. As we want to pick some commits to reduce number of patches. And it will happen quicker on develop, rather than on release.

Reply:
Sure, I'll update references and remove all patches when changes land in develop branch of dasharo repo.

`talos_2_support_payload` won't suit anymore, make it `talos_2` or do you have a better naming idea?

Reply:
talos_2 is fine!